Config.js

```diff
@@ -32,7 +32,7 @@ async function decrypt(
 async function create() {
   require('dotenv').config()
```

Comment:
Since you are setting dotenv config to envConfig, you could remove this. Also you can use this to catch errors in loading env variables if you want